config/spl-build.m4

```diff
@@ -1654,3 +1656,51 @@ AC_DEFUN([SPL_AC_KERNEL_READ], [
 	])
 	EXTRA_KCFLAGS="$tmp_flags"
 ])
+
+dnl #
+dnl # 4.15 API change
+dnl # https://lkml.org/lkml/2017/11/25/90
```

Comment:
The `timer_setup()` interface was introduced in 4.14 and the legacy interface retired in 4.15.  So let's rework this comment.

```
dnl #
dnl # 4.14 API change
dnl # timer_setup() interface added to replace the existing setup_timer().
dnl #
```